plugins/jfrog-artifactory/src/components/JfrogArtifactoryRepository/JfrogArtifactoryRepository.tsx

```diff
@@ -22,15 +22,17 @@ const useLocalStyles = makeStyles({
   },
 });
 
-export function JfrogArtifactoryRepository(props: RepositoryProps) {
+export function JfrogArtifactoryRepository(
+  props: RepositoryProps = { title: 'Docker Images' },
+) {
```

Comment:
how about below?

```suggestion
export function JfrogArtifactoryRepository(
 {widget , image, title = 'Docker images'}: RepositoryProps,
) {
```

Reply:
This looks better but if i made the following changes then i also need to change the below marked line:
```
export function JfrogArtifactoryRepository({widget,image,title = 'Docker Images'}: RepositoryProps,) {
  const jfrogArtifactoryClient = useApi(jfrogArtifactoryApiRef);
  const classes = useStyles();
  const localClasses = useLocalStyles();
  const [edges, setEdges] = useState<Edge[]>([]);


  const title = `Jfrog Artifactory repository: ${props.image}`; <=== need to remove this line due to Duplicate identifier 'title' or we need to remove the const like "title = `Jfrog Artifactory repository: ${image}`;". But as per my understanding we should not remove constant as it may cause issue in runtime.

  const { loading } = useAsync(async () => {


    const tagsResponse = await jfrogArtifactoryClient.getTags(image); <=== here 'props.image' needs to replace by 'image'

    setEdges(tagsResponse.data.versions.edges);

    return tagsResponse;
  });
```

Please let me know if my understanding is correct or not.

Reply:
yeah you would need to use values directly over `props.*`